### DISCUSSION TITLE
请问怎么自定义播放器js啊

### DISCUSSION THREAD

**Discussion by duskroot** [spencerwooo/onedrive-vercel-index#790] [Q&A] [answered]
可否替换html视频播放器？

**Comment by spencerwooo** [comment, marked as answer]
没太看懂你的意思。视频播放器使用的是 Plyr，是一个 React 组件，你可以自行更改 [components/previews/VideoPreview.tsx](https://github.com/spencerwooo/onedrive-vercel-index/blob/main/components/previews/VideoPreview.tsx) 里面的内容来自定义播放器渲染的方式。
